src/modules/m_filter.cpp

```diff
@@ -440,6 +440,7 @@ ModResult ModuleFilter::OnPreCommand(std::string &command, std::vector<std::stri
 			{
 				/* Note: We gline *@IP so that if their host doesnt resolve the gline still applies. */
 				GLine* gl = new GLine(ServerInstance->Time(), f->gline_time, ServerInstance->Config->ServerName.c_str(), f->reason.c_str(), "*", user->GetIPString());
+				ServerInstance->SNO->WriteGlobalSno('a', "FILTER: " + user->nick + " had their " + command + " message filtered and was G-Lined: " + f->reason);
```

Comment:
"Wibble had their PRIVMSG message filtered" sounds a bit weird to me?

Reply:
Will only happen for QUIT and PART.

Reply:
Yes, I misread. Thanks for clarifying.